kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/MobiComConversationFragment.java

```diff
@@ -321,6 +330,16 @@
     private View kmAutoSuggestionDivider;
     private String loggedInUserId;
 
+    FeedbackInputFragment feedBackFragment;
```

Comment:
Move all these to a seperate class. We cannot clutter anymore views in this fragment. Create a new View class and define these View there. Something like:

KmFeedBackView {
  
}

Only create the instance of this view in the fragment. If something needs to be accesed from fragment , provide a method on this view, if some callback is needed, create an interface for the same.

Also move all the methods related to this in the same class.

Reply:
@reytum Made the required changes.